### ISSUE TITLE
Libretime running with persistency on Docker (based on Ubuntu 18.04)

### ISSUE THREAD

**Issue by kessibi** [libretime/libretime#949]
Hello, I have recently been working on a way of getting Libretime to work within a docker container (a __single__ container for all applications of Libretime): https://github.com/kessibi/libretime-docker 

I am proud of saying that the installation works correctly and the image can be persistent if volumes are passed. The image still misses some improvement but I thought it was a nice enough proof of concept. Would such a container be what you'd have in the container repository (https://github.com/LibreTime/libretime-container-images) ?

It is possible to test the image with `docker run -p 80:80 -p 8000:8000 -p 8001:8001 --name=libretime odclive/libretime-docker:latest`. The whole install process is described in the README file.

I am fairly keen on getting some feedback from you on this.

PS: Sorry if the issue was not the best way to contact you, I thought it would be relevant enough.

**Comment by Robbt** [comment]
This looks like it solves #624 - I will test it out when I get a chance but this is definitely something we would be interested in integrating into the project. I didn't get very far in terms of working on a single docker container instance of LibreTime.

**Comment by paddatrapper** [comment]
I really like the concept! Some preliminary testing produced https://github.com/kessibi/libretime-docker/issues/6

**Comment by kessibi** [comment]
I do not have an extended knowledge of all the perks of Libretime so I depend on your feedback and the issues people provide me to improve the image.

Thanks for your consideration.

**Comment by paddatrapper** [comment]
As for the question about the container repository - I think this depends on the purpose of that repo. If it is for a production-ready solution, then @ned-kelly multi-container is a better fit. But, if we want that repo to be a simple easy to setup and evaluate LibreTime, then a single container would be a good fit

**Comment by Aphris-Karu** [comment]
> As for the question about the container repository - I think this depends on the purpose of that repo. If it is for a production-ready solution, then @ned-kelly multi-container is a better fit. But, if we want that repo to be a simple easy to setup and evaluate LibreTime, then a single container would be a good fit

You can do both. It does not have to be an ether or situation.

**Comment by Robbt** [comment]
Yeah at this point what we need people to test it and we can integrate it into our docs. It'd also be nice to have an official docker image supported by the project itself or to integrate one of the existing ones. This was supposed to happen with the multi-container one but we didn't coordinate the permissions for the transfer while @ned-kelly was still active and so it didn't happen. It could be revisited or we could create our own image based upon that.

One they are tested we can then just add it to the documentation.

**Comment by kessibi** [comment]
Has the libretime development stopped?
Is this issue still relevant to the overall project?
Thanks.

**Comment by paddatrapper** [comment]
LibreTime development has definitely not stopped and this is still relevant. The bot is automatic for all issues without much activity. I believe this issue is waiting on feedback from testing. If it is considered stable and works, I'd be happy moving the project to https://github.com/LibreTime/libretime-container-images and granting @kessibi permissions there. Once it is there, the docs will need updating to include it as a possible way of installing

**Comment by kessibi** [comment]
For the sake of keeping you informed, the image now supports version 3.0.0-alpha9 (and I switched to debian:buster).
https://github.com/kessibi/libretime-docker/tree/3.0.0-alpha9

(It may need some more testing, but POC is here.
